.storybook/main.mjs

```diff
@@ -16,18 +16,34 @@ export default {
 		getAbsolutePath('@storybook/addon-essentials'),
 		getAbsolutePath('@storybook/addon-a11y'),
 		getAbsolutePath('@storybook/addon-interactions'),
+		getAbsolutePath('@storybook/addon-coverage'),
 	],
-	webpackFinal: (config) => ({
-		...config,
-		resolve: {
-			...config.resolve,
-			alias: {
-				...config.resolve.alias,
-				'@storybook/blocks': fileURLToPath(new URL('../node_modules/@storybook/blocks', import.meta.url)),
-				'@storybook/docs-tools': fileURLToPath(new URL('../node_modules/@storybook/docs-tools', import.meta.url)),
+	webpackFinal: (config) => {
+		return {
+			...config,
+			module: {
+				...(config.module || {}),
+				rules: [
+					...(config.module.rules || []),
+					{
+						test: /\.(js|ts)$/,
+						loader: '@jsdevtools/coverage-istanbul-loader',
+						enforce: 'post',
+						include: join(process.cwd()),
```

Comment:
`cwd` can sometimes be tricky
Why don't we use `fileURLToPath(import.meta.url)`?

Reply:
Wouldn't `import.meta.url` will point to `node_modules`?

Reply:
no it points to the current file URL (it is the same as __filename in CSJ world)